signup.php

```diff
@@ -0,0 +1,43 @@
+
+
+<?php 
+
+include('config/init.php');
+echoHeader('Sign up');
+if(isset($_REQUEST['newSignUp'])){
+    insertUser(
```

Comment:
This looks like it should work, but a couple things you can do to polish this if you want more forms practice:

- redirect the user back to this page after they've submitted the form. This way, if they reload the page, it won't re-submit the form (thus creating duplicate users)
- You might consider doing some validation. Like, is the email address in the right format? Are all the fields set? Has someone with this email/username already signed up?
- A lot of signup forms these days use the email as the username (so people don't have to remember whatever username they picked). There are reasons not to do this (e.g. if the username will be displayed somewhere and it's sort of their personal alias) but it's something to consider.
- We'll talk about this in a lesson sometime, but just so you know, you shouldn't store passwords directly in the database (this is for security reasons). It's fine if you want to do this for now, but I'd make sure you don't ask anyone to actually sign up on your live website until you've implemented some extra security measures here.